pkg/scheduler/framework/plugins/mostrequestedpriority/most_requested.go

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mostrequestedpriority
+
+import (
+	"fmt"
+
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/kubernetes/pkg/scheduler/algorithm/priorities"
+	"k8s.io/kubernetes/pkg/scheduler/framework/plugins/migration"
+	framework "k8s.io/kubernetes/pkg/scheduler/framework/v1alpha1"
+)
+
+// MostRequestedPriority is a priority plugin that favors nodes with most requested resources.
+type MostRequestedPriority struct {
+	handle framework.FrameworkHandle
+}
+
+var _ = framework.ScorePlugin(&MostRequestedPriority{})
+
+// Name is the name of the plugin used in the plugin registry and configurations.
+const Name = "MostRequestedPriority"
```

Comment:
If we follow the pattern as `imagelocality` and `nodeaffinity`, we could name it to `MostRequested`. Then other places also do not need the `priority` suffix.

Reply:
done! PTAL